tests/test_redis.py

```diff
@@ -46,16 +49,17 @@ def test_random_port(redisdb_random):
     assert redisdb_random.keys('*') == []
 
 
-redis_proc_old_redis = factories.redis_proc(
-    executable='/usr/local/bin/redis-server',
-    port='?')
-redisdb_old_redis = factories.redisdb('redis_proc_old_redis')
+redis_proc_to_mock = factories.redis_proc(port='?')
 
 
-@pytest.mark.xfail(raises=RedisUnsupported)
-def test_old_redis(redisdb_old_redis):
+def test_old_redis(request):
```

Comment:
I suppose it's only to check if fixture will rise the RedisUnsupported exception. made it more explicit. We can parametrize this test with other possible versions, and I agreee the docs should be extended.

Made an issue for that: #132